### PR TITLE
Gw sprint 44

### PR DESCRIPTION
**Pull request by G-White-ISB**
add scripts to refactor image buckets

### CODE REVIEW

**Comment by bcli4d** [comment]
I'm only looking mv_image_files.py, and I'm not very familiar with the S3 libraries. Given that, is the intent of line 139:
`map_bucket_contents = s3_client.list_objects_v2(Bucket=map_bucket, Prefix=map_folder)['Contents']`
to get a list of series? And which is then expanded in move_some_blobs() to a list of instances in the series?
If so, I'd have expected that you need something like a Delimiter='/' parameter to list_buckets_v2. 
GCS, of course, has a similar function. In the GCS case, I think that if you do not specify a delimiter, you'd get all blob names that start with Prefix.
In other words, are you sure that you are distributing at series granularity to the workers?

**Comment by G-White-ISB** [comment]
No, line 139 is not getting a list of all series. It reads a list of parquet (table) files. Each table has a partition of the uuid_url_map_view view. The contents of each table are then the set of 'blobs' sent to a worker. But its distributing at image file granularity, not series granularity.

**Comment by bcli4d** [comment]
Thanks. As I said...I'm not up on S3.
Otherwise, mv_image_files.py looks good to me.
